scripts/ci/lib.sh

```diff
@@ -1328,12 +1332,15 @@ send_slack_failure_summary() {
         return 1
     fi
 
-    local slack_mention=""
-    _make_slack_mention
+    local mention_author=""
+    _do_we_at_the_author
```

Comment:
It took me a good while to understand what this means...
```suggestion
    _set_mention_author
```